core/src/main/java/com/scalar/db/api/AbacAdmin.java

```diff
@@ -0,0 +1,757 @@
+  }
+
+  /**
+   * Adds a compartment to a user with the given username for the given policy. Before adding the
+   * compartment, levels must be set to the user.
+   *
+   * @param policyName the policy name
+   * @param username the username
+   * @param compartmentShortName the short name of the compartment to set
+   * @param accessMode the access mode
+   * @param defaultCompartment whether the compartment is the default compartment
+   * @param rowCompartment whether the compartment is the row compartment
+   * @throws ExecutionException if the operation fails
+   */
+  default void addCompartmentToUser(
+      String policyName,
+      String username,
+      String compartmentShortName,
+      AccessMode accessMode,
+      boolean defaultCompartment,
+      boolean rowCompartment)
+      throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Removes a compartment from a user with the given username for the given policy.
+   *
+   * @param policyName the policy name
+   * @param username the username
+   * @param compartmentShortName the short name of the compartment to remove
+   * @throws ExecutionException if the operation fails
+   */
+  default void removeCompartmentFromUser(
+      String policyName, String username, String compartmentShortName) throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Adds a group to a user with the given username for the given policy. Before adding the group,
+   * levels must be set to the user.
+   *
+   * @param policyName the policy name
+   * @param username the username
+   * @param groupShortName the short name of the group to set
+   * @param accessMode the access mode
+   * @param defaultGroup whether the group is the default group
+   * @param rowGroup whether the group is the row group
+   * @throws ExecutionException if the operation fails
+   */
+  default void addGroupToUser(
+      String policyName,
+      String username,
+      String groupShortName,
+      AccessMode accessMode,
+      boolean defaultGroup,
+      boolean rowGroup)
+      throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Removes a group from a user with the given username for the given policy.
+   *
+   * @param policyName the policy name
+   * @param username the username
+   * @param groupShortName the short name of the group to remove
+   * @throws ExecutionException if the operation fails
+   */
+  default void removeGroupFromUser(String policyName, String username, String groupShortName)
+      throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Drops the user access for the given username for the given policy.
```

Comment:
Would it be more straightforward to disable the given user instead of dropping access from the given user?
(User access is not really defined, so I wasn't sure what it means.)

Reply:
@feeblefakie As discussed, will rename it to `dropUserTagInfoFromUser()`. Thanks.

Reply:
Fixed in 2c387a75f0db8f83fee32d78ed34462126e2ec82. Thanks.